app/services/fastly_vcl/whitelisted_params.rb

```diff
@@ -0,0 +1,65 @@
+module FastlyVCL
```

Comment:
Suggesting `Fastly::VCL` but it's definitely not a blocker :D

Reply:
That conflicts with the gem, it's already in use.

Reply:
ahahaha so sad :(